packaging/ubuntu-16.04/rules

```diff
@@ -90,6 +90,16 @@ override_dh_auto_build:
 	cd cmd && ( ./configure --prefix=/usr --libexecdir=/usr/lib/snapd $(VENDOR_ARGS))
 	$(MAKE) -C cmd all
 
+	# Generate the real systemd units out of the available templates
+	cat data/systemd/snapd.service.in | \
```

Comment:
We might consider putting the sed details into a Makefile in data/systemd - and then call `make -C data/systemd SNAPD_ENVIRONMENT_FILE=foo libexecdir=bar SNAP_MOUNTDIR=foobar` here. This avoid that we duplicate the sed rules in all the packaging we do.